.idea/$PRODUCT_WORKSPACE_FILE$

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Any ideas how it was commited? You have gitignore file but it seems not working, why?

Reply:
I think the gitignore does work. It happened because I first sent the task without gitignore. Then I included it via the second commit. So all the files of project were uploaded through the first commit.